db/migrate/20171211131211_create_pages.rb

```diff
@@ -0,0 +1,16 @@
+# Create new migration for pages table
+class CreatePages < ActiveRecord::Migration[5.1]
+  def change
+    create_table :pages do |t|
+      t.integer :page_type
```

Comment:
page belongs to company. 
add company_id

Reply:
I will add association when the entity company will merge to the main branch because I haven't company model in my branch

Reply:
OK